README.md

```diff
@@ -129,6 +129,58 @@ This is ideal if you are worried about accidentally exposing your metrics, are c
 
 The exporter runs by default on port `9394` and the metrics are available at the standard path of `/metrics`, the stand-alone exporter is configured to use gzip.
 
+
+### Rack Middleware
```

Comment:
nit: Jus thinking now, would it be more explicit to call it Rails Middleware in the doc? what about the class name?

```suggestion
### Rails Middleware
```